src/vos/tests/vts_io.h

```diff
@@ -71,7 +71,7 @@ struct io_test_args {
 	const char		*dkey;
 	const char		*akey;
 	void			*custom;
-	int			 ofeat;
+	enum daos_otype_t 	 ofeat;
```

Comment:
(style)  please, no space before tabs

Reply:
I just followed the format in this file. Please let me know if you do want me to remove the one space around this line. Thank you!